### PR TITLE
Add WRFDA build instruction

### PR DESCRIPTION
**Pull request by liujake**
Add instructions to build WRFPlus and WRFDA (4DVar mode and non-4Dvar mode).

### CODE REVIEW

File: README_user.md [hunk 1]
```diff
@@ -486,6 +486,99 @@ Diffing SERIAL/wrfout_d01_2000-01-24_12:00:00 wrfout_d01_2000-01-24_12:00:00
...
+-rwxr-xr-x 1 liuz ncar 64177872 Mar 12 10:56 wrfplus.exe
+```
+
+For WRFDA-4DVar build:
+```
+cd ~
+cp -pr WRF WRFDA
+cd WRFDA
+setenv CRTM 1   # will build with CRTM, optinal
+setenv RTTOV rttov-lib-directory # 3rd part software, optional
```
Comment (davegill) [inline README_user.md:515]: @liujake 
Jake,
This is not in the current container. Do we tell users how to get this (where to put it, how to build it) or just not include it at all?

**Comment by davegill** [comment]
@liujake 
Jake,
Thanks!

File: README_user.md (hunk 1 above)
Comment (liujake) [reply README_user.md:515]: CRTM is inside WRFDA package. RTTOV is optional and can be skipped for now. I can add more info about where to get RTTOV in the future when we include WRFDA test cases.

Comment (liujake) [reply README_user.md:515]: @davegill rttov line (and hdf5 line) can be removed if you like.

**Pull request merged**